src/components/CodeEditorWindow/config/Rules.js

```diff
@@ -15,7 +15,7 @@ export const Rules = {
 
             [/".*?"/, 'string'],
 
-            [/\/\//, 'comment'],
+            [/\/\/.*/, 'comment'],
```

Comment:
This will highlight the whole comment in a proper color. @kartik-gupta-ij

Reply:
Okay got it...